test/hardware/test_relay.py

```
@@ -139,22 +172,311 @@ async def test_multiple_filters():
+    except Exception as e:
+        print(f"FAIL: Ephemeral events - {e}")
+        failed += 1
+
+
+async def test_stored_events_before_eose():
+    global passed, failed
+    try:
+        unique_content = f"stored_test_{int(time.time())}"
+        async with websockets.connect(RELAY_URL) as ws:
+            event = make_event(unique_content, kind=1)
+            response = await send_recv(ws, ["EVENT", event])
```

Ruff (0.14.11)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)